server/src/db.js

```diff
@@ -1,11 +1,8 @@
-let user = process.env.DB_USER || process.env.USER;
-let pass = process.env.DB_PASS;
-let host = process.env.DB_HOST || "localhost:5432";
-
-pass = pass ? ":" + pass : "";
+const config = require("./config").root();
```

Comment:
`config.js` does export the Convict schema, but it's easy to overlook (`let conf = module.exports = convict({`). I'll move the `module.exports` assignment to the bottom, so that it's easier to follow. `.root()` just converts the exported schema into a POJO.